client/related_items/ItemsList.jsx

```diff
@@ -102,6 +103,23 @@ class ItemsList extends React.Component {
     if (!foundDuplicate) { this.setState({ outfits: outfits }); }
   }
 
+  removeFromOutfit(component) {
+    let storage = window.localStorage;
+
+    if (storage.length > 0) {
+      let currentOutfits = this.state.outfits;
+      let updatedOutfits = [];
+      let removedItemId = component.props.productInfo.id;
+
+      currentOutfits.forEach(outfit => {
```

Comment:
looks good. a nice use case for .filter() if you want to make this a little shorter.

Reply:
Ah yes good catch!